docs/Config.md

```diff
@@ -201,6 +201,7 @@ keybinding:
     fetchRemote: 'f'
   commits:
     squashDown: 's'
+    rebaseCommit: 'r' # only in branch commit view
```

Comment:
There should really be one keybinding for rebasing onto a ref. What are your thoughts on us just using `branches.rebaseBranch` given it already exists? We could create a new keybinding but I don't think it's really necessary.